data/json/monsters/zed_misc.json

```
@@ -1,4 +1,85 @@
 [
+  {
+    "id": "mon_frog_mother",
+    "type": "MONSTER",
+    "name": { "str": "frog mother" },
+    "description": "A large, bulbous creature with an extremely large mouth and small, stubby legs with no feet.  It is covered in an oily greenish-black residue and large, white pustules that seem ready to pop.",
+    "default_faction": "zombie",
+    "species": [ "ZOMBIE", "AMPHIBIAN" ],
+    "volume": "240 L",
+    "weight": "300 kg",
+    "hp": 600,
+    "speed": 75,
+    "material": [ "flesh" ],
+    "symbol": "F",
+    "color": "black_green",
+    "aggression": 100,
+    "morale": 100,
+    "melee_skill": 1,
+    "melee_dice": 1,
+    "melee_dice_sides": 4,
+    "melee_bash": 5,
+    "dodge": 2,
+    "armor_bash": 12,
+    "armor_cut": 6,
```

```suggestion
    "armor_cut": 2,
```
With the amount of meat she's got I think she should avoid the tough+armour meta, we have lots of that.